azurelinuxagent/common/cgroupapi.py

```diff
@@ -38,6 +39,12 @@ def create_agent_cgroups(self):
     def create_extension_cgroups_root(self):
         raise NotImplementedError()
 
+    def create_extension_cgroups(self):
```

Comment:
You're missing the parameter `extension_name` in the interface signature. The implementing methods use it.

Reply:
thank you, fixed